src/pug/guides/code/node/interactive/1.js

```diff
@@ -0,0 +1,7 @@
+const Beam = require('beam-client-node');
+const Tetris = require('beam-interactive-node');
+const rjs = require('robotjs');
+
+const channelId = 1234;
+const username = 'connor';
```

Comment:
You're correct, Its a force of habit from the many times over the past few months I've committed my actual beam password.